<launch>

  <node pkg="sonar" name="sonar_node_2" type="sonar_node_2" />
  <node pkg="sonar" name="sonar_handle_2" type="sonar_handle_2" />

  <node pkg="hokuyo_node" name="hokuyo" type="hokuyo_node" respawn="false" output="screen">
    <param name="calibrate_time" type="bool" value="false"/>
    <param name="port" type="string" value="/dev/ttyACM0"/>
    <param name="intensity" type="bool" value="false"/>
  </node>
</launch>

